ksql-engine/src/main/java/io/confluent/ksql/metrics/ProducerCollector.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.ksql.metrics;
+
+import io.confluent.common.metrics.stats.Rate;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import io.confluent.common.metrics.*;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class ProducerCollector {
+
+  final Map<String, Counter> topicCounters = new HashMap<>();
+  private final Metrics metrics;
+  private String id;
```

Comment:
This probably should be named `clientId`, because that's what it seems to be set to.